src/components/HomePage/Event.js

```diff
@@ -57,9 +75,8 @@ function Event({ startTime, name, location, description }) {
 }
 
 Event.propTypes = {
-	startTime: PropTypes.instanceOf(Date).isRequired,
 	name: PropTypes.string.isRequired,
-	location: PropTypes.string.isRequired,
+	subtitles: PropTypes.array.isRequired,
```

Comment:
nit:
```suggestion
	subtitles: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
```

Ideally we would make this component work even if subtitles is not supplied, but since we don't need that capability right now it's fine to make `subtitles` a required prop.